filebeat/module/postgresql/_meta/kibana/7/dashboard/Filebeat-Postgresql-slowlogs.json

```diff
@@ -169,7 +169,7 @@
                         "index": "filebeat-*", 
                         "query": {
                             "language": "lucene", 
-                            "query": "postgresql.log.duration:>30"
+                            "query": "event.duration:>30"
```

Comment:
@webmat Did we change the unit here? if yes, we probably need to adjust this query?

Reply:
Scale change was x1000000, so the dashboard needs the same adjustment.

Reply:
I will follow up on these in a separate PR to adjust it.